src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -299,8 +299,7 @@
 									<ColumnDefinition Width="Auto" />
 									<ColumnDefinition
 										x:Name="Column2"
-										Width="{x:Bind ColumnsViewModel.NameColumn.Length, Mode=OneWay}"
-										MaxWidth="{x:Bind ColumnsViewModel.NameColumn.MaxLength, Mode=OneWay}" />
```

Comment:
Should `MaxLength` be removed from the ViewModel?

Reply:
Personally, I would remove this constraint. I don't think it's a problem if the user has very wide columns.

Reply:
Most columns should have this constraint, NameColumn is the exception. Is it possible to set the default for the NameColumn MaxLength to *?

Reply:
We can set `double.MaxValue` as default value

Reply:
We don't actually have to bind to it in the xaml but I want the c# value to reflect it.

Reply:
What about something like this:

        private ColumnViewModel nameColumn = new ColumnViewModel()
        {
            NormalMaxLength = double.PositiveInfinity
        };

Reply:
Maybe just set the max width to 1000 😂